common/utils.go

```diff
@@ -49,6 +50,14 @@ func AddressHex(address common.Address) string {
 	return Hex(address.Hex())
 }
 
+func StringToAddress(s string) common.Address { return BytesToAddress([]byte(s)) }
```

Comment:
Why not just use `common.HexToAddress` in geth `common`?

Reply:
replace it with `ethereum/common.BytesToAddress()` 😆